oci/private/image.bzl

```diff
@@ -171,7 +171,7 @@ def _oci_image_impl(ctx):
         args.add(ctx.file.base.path, format = "--from=%s")
         inputs.append(ctx.file.base)
         if use_symlinks:
-            transitive_inputs.append(ctx.file.base)
+            transitive_inputs += [ctx.file.base] + ctx.attr.base[DefaultInfo].default_runfiles.files.to_list()
```

Comment:
I think you can avoid a `to_list` here since `ctx.actions.run#inputs` accepts a depset

Reply:
right, fixed it now.